.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
```

Comment:
If we add a gitignore I'd go with this one personally
https://github.com/github/gitignore/blob/master/Python.gitignore